### PR TITLE
Add cache for get job and get translations requests

### PR DESCRIPTION
**Pull request by hadomskyi**
ENG-12408
ENG-12365

### CODE REVIEW

File: src/services/repositories/external/ConnectorJobRepository.php [hunk 1]
```diff
@@ -68,6 +70,62 @@ public function start(int $liltJobId): bool
      */
     public function findOneById(int $liltJobId): JobResponse
     {
-        return $this->apiInstance->servicesApiJobsGetJobById($liltJobId);
+        $cacheKey = __METHOD__ . ':' . $liltJobId;
+        $cache = CraftliltpluginParameters::getResponseCache();
+
+        if (!empty($cache)) {
```
Comment (hanibhat) [inline src/services/repositories/external/ConnectorJobRepository.php:76]: There is already a check for this in `getResponseCache()`. Do we need another one here?

File: tests/.env.test [hunk 2]
```diff
@@ -11,4 +11,5 @@ DB_PORT="3306"
 SECURITY_KEY="abcde12345"
 
 CRAFT_LILT_PLUGIN_CONNECTOR_API_KEY="SECURE_API_KEY_FOR_LILT_CONNECTOR"
-CRAFT_LILT_PLUGIN_CONNECTOR_API_URL="http://wiremock/api/v1.0"
\ No newline at end of file
+CRAFT_LILT_PLUGIN_CONNECTOR_API_URL="http://wiremock/api/v1.0"
+CRAFT_LILT_PLUGIN_CACHE_RESPONSE_IN_SECONDS=0
```
Comment (hanibhat) [inline tests/.env.test]: Needs a line break

**Review by hanibhat — APPROVED** [review]
Left small comments but LGTM

File: src/services/repositories/external/ConnectorJobRepository.php (hunk 1 above)
Comment (hadomskyi) [reply src/services/repositories/external/ConnectorJobRepository.php:76]: @hanibhat check in `getResponseCache` is for getting value as 0 or N (number of seconds). But now if we have `0` it means cache is disabled and `1 or more` means using cache.

File: tests/.env.test (hunk 2 above)
Comment (hadomskyi) [reply tests/.env.test]: 👍

**Review by hanibhat — APPROVED** [review]

**Pull request merged**